src/VmaReplay/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# This file will only be executed if VMA_BUILD_REPLAY is set to ON
+
+if (WIN32)
+    set(VMA_REPLAY_SOURCE_FILES
+        Common.cpp
+        Constants.cpp
+        VmaReplay.cpp
+        VmaUsage.cpp
+    )
+
+    add_executable(VmaReplay ${VMA_REPLAY_SOURCE_FILES})
```

Comment:
VmaSample executable has /MP option used while this one doesn't. I think we should configure both the same way.

Reply:
Done.